lib/utils/src/graph/adjacency_multidigraph.cc

```diff
@@ -10,6 +10,16 @@ Node AdjacencyMultiDiGraph::add_node() {
   return node;
 }
 
+NodePort AdjacencyMultiDiGraph::add_node_port() {
+  NodePort nodePort{this->next_node_port};
```

Comment:
```suggestion
  NodePort nodePort = {this->next_node_port};
```
I generally prefer this: see https://abseil.io/tips/88

Reply:
it will meet error.

```
error: converting to ‘FlexFlow::NodePort’ from initializer list would use explicit constructor ‘FlexFlow::NodePort::NodePort(const long unsigned int&) [inherited from FlexFlow::strong_typedef<FlexFlow::NodePort, long unsigned int>]’
   NodePort nodePort={this->next_node_port};
```